categories/categories.yml

```diff
@@ -0,0 +1,143 @@
+categories:
+  aws:
+    title: "AWS"
+  azure:
+    title: "Azure"
+  cloud:
+    title: "Cloud"
+  config_management:
+    title: "Config management"
+  containers:
+    title: "Containers"
+  crm:
+    title: "CRM"
+  custom:
+    title: "Custom"
+  datastore:
+    title: "Datastore"
+  elastic_stack:
+    title: "Elastic Stack"
+  enterprise_serch:
+    title: "Enterprise Search"
+    subcategories:
+      app_search:
+        title: "Application Search"
+      connector: # TODO: Do we want three different connector categories?
```

Comment:
@mukeshelastic @nimarezainia would you be able to weight in here? Do we want three categories?

Reply:
yes, let's limit the subcategories to only one level depth.. category -> subcategory. And author can specify only one subcategory

Reply:
> And author can specify only one subcategory

Does it mean that a package can have only a subcategory? We don't have this limitation with categories, and this may complicate things.